tests/plugins_test.go

```diff
@@ -150,7 +150,7 @@ func TestGeneration(t *testing.T) {
 					testPluginWithImage(t, toTest, "grpc-federation")
 				}
 				if semver.Compare(toTest.PluginVersion, "v0.11.0") >= 0 {
-					testPluginWithImage(t, toTest, "grpc-federation-v0.11.0")
+					testPluginWithImage(t, toTest, "grpc-federation-latest")
```

Comment:
On second thought, maybe we should just keep all existing up-to the newly built image and not worry about anything breaking.

Reply:
added explicit image for v0.13.6 and higher.